### PR TITLE
First commit

### PR DESCRIPTION
**Pull request by NathanMcGee**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*
*
*

### CODE REVIEW

File: 101-vpn-custom-ipsec-policy/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,203 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "location" : {
```
Comment (bmoore-msft) [inline 101-vpn-custom-ipsec-policy/azuredeploy.json:5]: add a metadata description to the location param

File: 101-vpn-custom-ipsec-policy/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+        },
+        "localGateway_Name": {
+            "type": "string",
+            "metadata": {
+                "description": "Name of existing Local Network Gateway to deploy the connection to"
+            }
+        },
+        "vpnName": {
+            "type": "string",
+            "metadata": {
```
Comment (bmoore-msft) [inline 101-vpn-custom-ipsec-policy/azuredeploy.json:23]: move all the metadata properties to the bottom of the param declaration

File: 101-vpn-custom-ipsec-policy/metadata.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+    "$schema": "https://aka.ms/azure-quickstart-templates-metadata-schema#",
+    "itemDisplayName": "101 - VPN Custom IPSec Policy",
```
Comment (bmoore-msft) [inline 101-vpn-custom-ipsec-policy/metadata.json:3]: remove the 101 from the display name

**Review by bmoore-msft — CHANGES_REQUESTED** [review]

File: 101-vpn-custom-ipsec-policy/azuredeploy.json (hunk 1 above)
Comment (NathanMcGee) [reply 101-vpn-custom-ipsec-policy/azuredeploy.json:5]: addressed in commit 604d1ab

File: 101-vpn-custom-ipsec-policy/azuredeploy.json (hunk 2 above)
Comment (NathanMcGee) [reply 101-vpn-custom-ipsec-policy/azuredeploy.json:27]: addressed in commit 604d1ab

File: 101-vpn-custom-ipsec-policy/metadata.json (hunk 3 above)
Comment (NathanMcGee) [reply 101-vpn-custom-ipsec-policy/metadata.json]: addressed in commit 604d1ab

**Pull request merged**